File: Boca/JAVA_2/Q5/src/CD.java
public class CD extends Produto {
    private int numFaixas;

    public CD (int codigoDeBarras, String nome, double preco, int numFaixas) {
        super(codigoDeBarras, nome, preco);
        this.numFaixas = numFaixas;
    }

    public int getNumFaixas() {
        return numFaixas;
    }


}

File: Boca/CPP/06/CD.cpp
#include "CD.hpp"

CD::CD(const string& nome, double preco, int numeroFaixas) : Produto(nome, preco), numeroFaixas(numeroFaixas) {}

void CD::print(ostream &out) const {
    Produto::print(out);
    out << " - " << numeroFaixas << " faixas";
}

ostream& operator<< (ostream &out, CD &cd) {
    cd.print(out);
    return out;
}

CD::~CD() {}